index.md

```diff
@@ -47,6 +47,7 @@ and submit a pull request.
 | [Bundler](https://bundler.io/) | `bundle COMMAND --no-color` ([Docs](https://bundler.io/v1.15/man/bundle.1.html)) |
 | [Clang](https://clang.llvm.org/) | `-fno-color-diagnostics` ([Docs](https://clang.llvm.org/docs/UsersManual.html#formatting-of-diagnostics)) |
 | [Cocoapods](https://cocoapods.org/) | `pod COMMAND --no-ansi` ([Docs](https://guides.cocoapods.org/terminal/commands.html#pod_install)) |
+| [RSpec](http://rspec.info/) | `rspec --no-color` | Can also be set in the `SPEC_OPTS` environment variable or the `.rspec` configuration file.
```

Comment:
This has an extra column that doesn't show up.  I think I would prefer an environment variable example above all others so it can be permanent and doesn't need to be invoked every time with a command-line flag.

Reply:
You got it.